x-pack/plugins/fleet/server/services/epm/agent/agent.ts

```diff
@@ -111,11 +111,12 @@ function buildTemplateVariables(variables: PackagePolicyConfigRecord, templateSt
   return { vars, yamlValues };
 }
 
-function containsHelper(this: any, item: string, list: string[], options: any) {
-  if (Array.isArray(list) && list.includes(item)) {
+function containsHelper(this: any, item: string, check: string | string[], options: any) {
+  if ((Array.isArray(check) || typeof check === 'string') && check.includes(item)) {
     if (options && options.fn) {
       return options.fn(this);
     }
+    return true;
```

Comment:
I do not think we need to return true, actually if we type correctly that function it should be something like this
```js
function containsHelper(
  this: any,
  item: string,
  list: string[],
  options: Handlebars.HelperOptions
) {
  if (Array.isArray(list) && list.includes(item)) {
    return options.fn(this);
  }
  return '';
}
```

Reply:
the `true` allows for the helper to be used in an `if` condition as shown in the tests. Basically if it's not used as a block-level helper directly (i.e. `{{#contains "..." variable}}content in here{{/contains}}`), you have to have the `true` to chain with an `if` (i.e. `{{if (contains "..." variable)}}something{{else}}other{{/if}}`).

Reply:
@nchaulet are we good on keeping this line as is?